tests/integration/linux_bridge_test.py

```diff
@@ -397,6 +397,111 @@ def test_port_vlan_not_implemented(port0_up):
         pass
 
 
+@pytest.mark.xfail(
+    reason='https://nmstate.atlassian.net/browse/NMSTATE-230', strict=True
+)
+class TestVlanFiltering(object):
+    access_tag = 300
+    bridge_name = TEST_BRIDGE0
+
+    @pytest.mark.parametrize(
+        'is_native_vlan', [True, False], ids=['native', 'not-native']
+    )
+    def test_trunk_port_config(self, is_native_vlan, port0_up):
+        trunk_tags = self._generate_vlan_id_config(100, 101)
```

Comment:
You declared `_generate_vlan_id_config` as static, but have called it through `self`.
That is technically possible, but very confusing to the reader. Please use `TestVlanFiltering` instead.

Reply:
Done.